express/code/blocks/how-to-v2/how-to-v2.css

```
@@ -0,0 +1,207 @@
+main .how-to-v2 h2 {
```

I would suggest scoping to the block name unless otherwise needed.
 
In general you shouldn't need to scope specifically to `main` unless some specific overrides are needed. Same for most of the styles below.
 
 ```suggestion
.how-to-v2 h2 {
```